---a/net/minecraft/network/EnumPacketDirection.java
+++b/net/minecraft/network/EnumPacketDirection.java
@@ -1,10 +1,8 @@
 package net.minecraft.network;
 
-public enum EnumPacketDirection
-{
+public enum EnumPacketDirection {
     SERVERBOUND,
     CLIENTBOUND;
 
     private static final String __OBFID = "CL_00002307";
-}
+}
